### PR TITLE
Feat/add bitget wallet support

### LINKED ISSUE #909: Add Bitget Wallet
## Description 

Following [this injection](https://github.com/lfglabs-dev/app.starknet.id/blob/testnet/utils/connectorWrapper.ts) model please add Bitget wallet support to Starknet Quest.

### PR DESCRIPTION
**Pull request by jaiminRaiyani**
This PR introduces support for the Bitget wallet in the Starknet Quest application. This includes updates to components and hooks to facilitate Bitget wallet integration and make it available for user selection.

**Features :**

app/provider.tsx:
Added a new InjectedConnector instance for Bitget wallet, making it selectable from the available connectors list.

components/UI/changeWallet.tsx:
Modified wallet change functionality to include the Bitget wallet in relevant button displays.
Updated wallet selection conditions to recognize Bitget, ensuring it appears alongside existing wallets like Braavos and ArgentX.

components/UI/iconsComponents/icons/walletIcons.tsx:
Added an SVG icon for Bitget wallet, ensuring consistent UI representation within the app.

hooks/useGetDiscoveryWallets.ts:
Enhanced the useGetDiscoveryWallets hook to include Bitget wallet discovery and download link for easy access.

Resolves: #909 

## Other information
the svg icon for bitget might not work so do not consider that

## Release Notes

- **New Features**
  - Added support for the "Bitget Wallet" connector, enhancing wallet connectivity options.
  - Updated the "Change Wallet" component to include the "bitkeep" connector for wallet installation.
  - Introduced a new icon for the "Bitget Wallet" in the wallet icons display.
  - Added functionality to retrieve the download link for the Bitkeep wallet.

- **Style**
  - Improved formatting of the `Typography` components for better readability.

- **Chores**
  - Cleaned up the configuration syntax in `next.config.js` without impacting functionality.

### CODE REVIEW

File: components/UI/changeWallet.tsx [hunk 1]
```diff
@@ -71,7 +78,11 @@ const ChangeWallet: FunctionComponent<ChangeWalletProps> = ({
               </div>
             );
           } else {
-            if (connector.id === "braavos" || connector.id === "argentX") {
+            if (
+              connector.id === "braavos" ||
+              connector.id === "argentX" ||
+              connector.id === "bitkeep"
+            ) {
```
[the remark this answers is not in the document]
Comment (Marchand-Nicolas) [reply components/UI/changeWallet.tsx]: @jaiminRaiyani this is a good suggestion https://github.com/lfglabs-dev/starknet.quest/pull/915/files#r1820659210

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]

File: components/UI/changeWallet.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jaiminRaiyani) [reply components/UI/changeWallet.tsx]: @Marchand-Nicolas  i have implimented this suggestion you can review it

**Comment by Marchand-Nicolas** [comment]
Hello @jaiminRaiyani Please resolve the conflicts

**Review by Marchand-Nicolas — APPROVED** [review]
Lgtm

**Comment by Marchand-Nicolas** [comment]
@jaiminRaiyani Waiting for conflicts to be resolved in order to merge

**Comment by jaiminRaiyani** [comment]
@Marchand-Nicolas  please review it i have resolved the conflict

**Review by Marchand-Nicolas — APPROVED** [review]
Lgtm :)

**Pull request merged**